src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/14.diff.sql

```diff
@@ -0,0 +1,550 @@
+/*************************************************************
+    Resource change capture feature
+**************************************************************/
+
+/*************************************************************
+    Resource change data table
+**************************************************************/
+
+IF NOT EXISTS (
+    SELECT 1 
+    FROM sys.tables
+    WHERE name = 'ResourceChangeData')
+BEGIN
+    CREATE TABLE dbo.ResourceChangeData
+    (
+        Id bigint IDENTITY(1,1) NOT NULL,
+        Timestamp datetime2(7) NOT NULL,
+        ResourceId varchar(64) NOT NULL,
+        ResourceTypeId smallint NOT NULL,
+        ResourceVersion int NOT NULL,
+        ResourceChangeTypeId tinyint NOT NULL,
+     CONSTRAINT PK_ResourceChangeData PRIMARY KEY CLUSTERED 
+    (
+        [Id] ASC
+    )WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON, OPTIMIZE_FOR_SEQUENTIAL_KEY = OFF) ON [PRIMARY]
+    ) ON [PRIMARY]
```

Comment:
nit: I feel that having consistent indentation and style would make it easier to read the code (this file and 14.sql file as well). We also have a style guide (that is linked to at the top of 14.sql) and it might be helpful to follow that.

A couple of examples: this block ends with two ON [PRIMARY] sentences and it's not very intuitive to what block they apply to. In other places, sometimes `(` starts on the same line, in other cases it starts on the next line.

Reply:
Updated the style.